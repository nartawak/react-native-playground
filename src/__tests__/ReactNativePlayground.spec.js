import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import App from '../ReactNativePlayground';

it('renders ReactNativePLayground application', () => {
  const component = () => {
    renderer.create(
      <App />
    ).toJSON();
  };
  expect(component).toMatchSnapshot();
});
