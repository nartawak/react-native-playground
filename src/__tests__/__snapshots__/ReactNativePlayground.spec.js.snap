exports[`test renders ReactNativePLayground application 1`] = `[Function]`;
